### PR TITLE
chore: update gitignore

### PR DESCRIPTION
**Pull request by alexindevs**
# Description
I added the configuration file to the `.gitignore`.

# Changes proposed

## What were you told to do?

Add `appsettings.json` file to `.gitignore` to fix the deployment error.

## What did you do?

- Added the `appsettings.json` file to the `.gitignore` file to ensure sensitive information is not pushed to the repository.
- Ensured that the change prevents deployment errors caused by misconfiguration in the `appsettings.json` file.

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/Videos

N/A

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -374,6 +374,8 @@ src/Hng.Web/internal-nlog.txt
 
 # AppSettings
 appsettings.*.json
+appsettings.json
+src/Hng.Web/appsettings.json
```
Comment (Godhanded) [inline .gitignore:378]: Don't include app settings in git ignore.

Use it to know what env variables are required and add them as envirot variables on the server.

The format is
__ for every level down a tree

**Review by Godhanded — CHANGES_REQUESTED** [review]
You don't put app settings in gitignore

File: .gitignore (hunk 1 above)
Comment (chimaihueze) [inline .gitignore:378]: Remove this from gitignore. Leave it as it is. You can create another file and pass the values through it, but not this app settings file.

**Review by chimaihueze — CHANGES_REQUESTED** [review]

**Comment by Godhanded** [comment]
Environment Variables should be used

**Pull request closed without merging**